### PR TITLE
CHK-435: Session token

### PR DESCRIPTION
**Pull request by wdsrocha**
#### What problem is this solving?

Resolves query to retrieve the session token and handle when session tokens are used on search arguments.

Depends on https://github.com/vtex-apps/geolocation-graphql-interface/pull/6.

#### How should this be manually tested?

[Run this GraphQL query.](https://w0geo--checkoutio.myvtex.com/_v/private/vtex.geolocation-graphql-interface@0.1.0/graphiql/v1?query=%7B%0A%20%20sessionToken%0A%7D%0A)

The session token handling part is quite hard to test, as there is no easy way to tell that the session tokens are being used correctly. Check the Jira ticket comment for ways to test it in the long run.

#### Checklist/Reminders

- [x] Linked this PR to a JIRA story (if applicable).
- [ ] Deleted the workspace after merging this PR (if applicable).

#### Notes

[Google suggests using UUID](https://developers.google.com/places/web-service/session-tokens#create-session-tokens), but any unique token would be okay as well.

Also, I suggest reviewing per commit.

### CODE REVIEW

**Comment by wdsrocha** [comment]
Going to add tests soon 👀

**Comment by wdsrocha** [comment]
Somehow linter is broken 🤔 but will be fixed once the dependency PR gets merged.

File: node/__tests__/sessionToken.test.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import getSessionToken from '../resolvers/sessionToken'
+
+describe('sessionToken', () => {
```
Comment (wdsrocha) [inline node/__tests__/sessionToken.test.ts:3]: Not even sure if it's worth to add those tests tbh

**Review by jeffersontuc — APPROVED** [review]
LGTM

**Review by lucasecdb — APPROVED** [review]

**Pull request merged**